OrbitLinuxTracing/PerfEvent.h

```diff
@@ -166,9 +166,8 @@ class LostPerfEvent : public PerfEvent {
   uint32_t GetCpu() const { return ring_buffer_record.sample_id.cpu; }
 };
 
-struct __attribute__((__packed__)) dynamically_sized_perf_event_stack_sample {
-  struct __attribute__((__packed__))
-  dynamically_sized_perf_event_sample_stack_user {
+struct dynamically_sized_perf_event_stack_sample {
```

Comment:
That's a potential bug which hasn't been reported by clang.

Reply:
How is this a potential bug?

Reply:
The packed attribute only applies to POD-structs. Since dynamically_sized_perf_event_sample_stack_user contains a unique_ptr it's not POD anymore so the __packed__ is (silently) ignore.

Since we don't need the __packed__ in this case the fix was easy and that's why it's only a "potential" bug.